src/chrome/content/rules/Threema.xml

```diff
@@ -1,27 +1,26 @@
 <ruleset name="Threema">
-    <target host="threema.ch" />
-    <target host="www.threema.ch" />
-    <target host="gateway.threema.ch" />
-    <target host="shop.threema.ch" />
-    <target host="myid.threema.ch" />
-    <target host="avatar.threema.ch" />
-    <target host="work.threema.ch" />
-    <target host="three.ma" />
-    <target host="web.threema.ch" />
-    <target host="web-beta.threema.ch" />
+	<target host="threema.ch" />
+	<target host="www.threema.ch" />
+	<target host="gateway.threema.ch" />
+	<target host="shop.threema.ch" />
+	<target host="myid.threema.ch" />
+	<target host="avatar.threema.ch" />
+	<target host="work.threema.ch" />
+	<target host="three.ma" />
```

Comment:
Please don't interleave different domains. This should be after (or before) all `threema.ch` subdomains.

Reply:
Also sort the subdomains alphabetically